src/mame/hp/hp9825.cpp

```diff
@@ -131,29 +138,21 @@ class hp9825_state : public driver_device
 		, m_tape(*this , "tape")
 		, m_io_key(*this , "KEY%u" , 0)
 		, m_shift_key(*this , "KEY_SHIFT")
-		, m_scroll_key(*this , "KEY_SCROLL")
-		, m_prt_alpha_out(*this , "prt_alpha")
-		, m_prt_graph_out(*this , "prt_graph")
 		, m_prt_timer(*this , "prt_timer")
 		, m_beeper(*this , "beeper")
 		, m_beep_timer(*this , "beep_timer")
 		, m_io_slot(*this, "slot%u", 0U)
-		, m_prt_out(*this, "printer")
 		, m_display(*this , "char_%u_%u" , 0U , 0U)
 		, m_run_light(*this , "run_light")
 		, m_shift_lock(*this , "shift_lock")
 		, m_tape_led(*this , "tape_led")
 		, m_cassette(*this , "cassette")
-		, m_printer_bitmap(PRINT_OUT_W , PRINT_OUT_H)
 	{
 	}
 
-	void hp9825_base(machine_config &config);
+	void hp98xx_base(machine_config &config);
```

Comment:
If this is abstract, the constructor and machine configuration member should be protected.